Formula/kubevela.rb

```diff
@@ -18,18 +18,20 @@ class Kubevela < Formula
   depends_on "go" => :build
 
   def install
+    ENV["CGO_ENABLED"] = "0"
+    ENV["GO111MODULE"] = "on"
     ldflags = %W[
       -s -w
       -X github.com/oam-dev/kubevela/version.VelaVersion=#{version}
       -X github.com/oam-dev/kubevela/version.GitRevision=#{Utils.git_head}
-    ].join(" ")
+    ]
 
-    system "go", "build", *std_go_args(ldflags: ldflags), "-o", bin/"vela", "./references/cmd/cli"
+    system "go", "build", *std_go_args(output: bin/"vela", ldflags: ldflags), "./references/cmd/cli"
   end
 
   test do
     # Should error out as vela up need kubeconfig
-    status_output = shell_output("#{bin}/vela up 2>&1", 1)
+    status_output = shell_output("#{bin}/vela version 2>&1", 1)
```

Comment:
We should check that `vela version` produces the correct version number too.

Reply:
`get kubeConfig err invalid configuration: no configuration has been provided`  This error message is returned every time due to a built-in check.

Reply:
Is there no way to get `vela` to show the version number?

Reply:
Maybe I can try.